cypress/fixtures/constants.js

```diff
@@ -151,28 +151,54 @@ const TLeditDrawerBtn = "";
 const TLeditDrawerMsg = "";
 const TLdelete =
   "[data-row-key='0'] > :nth-child(4) > :nth-child(1) > :nth-child(3)";
+const TLdeleteYes = ".ant-btn-primary";
+const TLdeleteNo = ".ant-popover-buttons > :nth-child(1)";
 const TLpageNumberGroup = ".ant-pagination";
+const TLpageNext = ".ant-pagination-next > .ant-pagination-item-link";
+const TLpagePrevious = ".ant-pagination-prev > .ant-pagination-item-link";
 // CREDIT GROUP ADD
-const menuCreditGroupAdd = ":nth-child(7) > a";
+const menuCreditGroupAdd = ":nth-child(8) > a";
+const CGAbody = ".ant-card-body";
 const CGAcampus = ".ant-select-selection";
+const CGAcampusMsg =
+  ":nth-child(1) > .ant-col-20 > .ant-form-item-control > .ant-form-explain";
 const CGAname = "#name";
+const CGAnameMsg =
+  ":nth-child(2) > .ant-col-20 > .ant-form-item-control > .ant-form-explain";
 const CGAcancel = "[type='button']";
+const CGAcancelYes = ".ant-popover-buttons > .ant-btn-primary";
+const CGAcancelNo = ".ant-popover-buttons > :nth-child(1)";
 const CGAaddTeam = ".ant-btn-primary";
+const CGAconfirmation = ".ant-message-notice-content";
 // CREDIT GROUPS LIST
-const menuCreditGroupsList = ":nth-child(8) > a";
+const menuCreditGroupsList = ":nth-child(9) > a";
 const CGLname = ".ant-input";
+const CGLbody = ".ant-card-body";
 const CGLcampus = ".ant-select-selection";
 const CGLview =
   "[data-row-key='0'] > :nth-child(3) > :nth-child(1) > :nth-child(1)";
+const CGLviewDrawer = ".ant-drawer-wrapper-body";
 const CGLedit =
   "[data-row-key='0'] > :nth-child(3) > :nth-child(1) > :nth-child(3)";
+const CGLeditDrawer = ".ant-drawer-body";
+const CGLeditName = "#name";
+const CGLeditCancel = ".ant-form-item-children > [type='button']";
+const CGLeditSave = ".ant-btn-primary";
 const CGLteamMembership =
   "[data-row-key='0'] > :nth-child(3) > :nth-child(1) > :nth-child(5)";
+const CGLteamMembershipSave =
+  ".ant-col-24 > .ant-row > .ant-col > .ant-form-item-control > .ant-form-item-children > .ant-btn-primary";
```

Comment:
This can be made a variable `.ant-col-24 > .ant-row > .ant-col > .ant-form-item-control > .ant-form-item-children >`, use DRY principal